packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.html

```diff
@@ -365,6 +365,15 @@
                             </span>
                         </div>
                     </div>
+                    <div class="oui-tile__definition">
+                        <div class="oui-tile__definition">
+                            <strong class="oui-tile__term d-block" data-translate="overthebox_detail_links">
+                            </strong>
+                            <oui-tile-button data-ng-href="{{:: $ctrl.guidesLink }}" external>
+                                <span data-translate="overTheBox_tabs_docs"></span>
```

Comment:
Following translation located here:
- https://github.com/ovh/manager/blob/master/packages/manager/modules/overthebox/src/overthebox/translations/Messages_fr_FR.json#L8

Can be moved right next to `overthebox_detail_links` here:
- https://github.com/ovh/manager/blob/master/packages/manager/modules/overthebox/src/overthebox/details/translations/Messages_fr_FR.json

And I think also the translation key could be renamed since it's no more a Tab item.

Does that make sense for you?

Reply:
Not exactly, `overthebox_detail_links` is to display text "Links" before display the list of available links for OverTheBox. I could rename `overTheBox_tabs_docs` to access to guides